terraform/scripts/manually-upload-certs.sh

```diff
@@ -14,26 +16,23 @@ pass "certs/${AWS_ACCOUNT}/${DEPLOY_ENV}/apps_domain_intermediate.crt" > /dev/nu
 WORKING_DIR=$(mktemp -d cf-certs.XXXXXX)
 trap 'rm -r "${WORKING_DIR}"' EXIT
 
-if ! aws s3 ls "s3://${DEPLOY_ENV}-state/cf-certs.tfstate" --summarize | grep -q "Total Objects: 0"; then
-  aws s3 cp "s3://${DEPLOY_ENV}-state/cf-certs.tfstate" "${WORKING_DIR}/cf-certs.tfstate"
-else
-  echo "No previous cf-certs.tfstate file found in s3://${DEPLOY_ENV}-state/. Assuming first run."
-fi
+# Work in tmp dir to ensure there's no local state before we kick off terraform, it prioritises it
+PAAS_CF_DIR=$(pwd)
+cd ${WORKING_DIR}
```

Comment:
The `cd` here means that the above exit trap no longer works because `WORKING_DIR` doesn't contain an absolute path.